### PR TITLE
[@types/auth0] fix colors in branding

### PR DESCRIPTION
**Pull request by MoranJustt**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by nhardy — APPROVED** [review]

**Review by GalDayan — APPROVED** [review]
LGTM

File: types/auth0/index.d.ts [hunk 1]
```diff
@@ -1197,7 +1197,7 @@ export interface Organization {
     branding?:
         | {
               logo_url?: string | undefined;
-              colors: {
+              colors?: {
                   primary: string;
                   page_background: string;
               };
```
Comment (peterblazejewicz) [inline types/auth0/index.d.ts:1203]:
```suggestion
              } | undefined;
```

File: types/auth0/index.d.ts [hunk 2]
```diff
@@ -1216,7 +1216,7 @@ export interface CreateOrganization {
...
```
Comment (peterblazejewicz) [inline types/auth0/index.d.ts:1222]:
```suggestion
              } | undefined;
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Review by nhardy — APPROVED** [review]

**Comment by MoranJustt** [comment]
> @MoranJustt: Everything looks good here. I am ready to merge this PR (at [d1af3b2](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d1af3b20ee0830e4a214dc94340bcc419af8d617)) on your behalf whenever you think it's ready.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! ❤️
> 
> (@ianhowe76, @pwrnrd, @CatGuardian, @MengRS, @Scoup, @isabela-morais, @rbutera, @peterblazejewicz, @danursin, @nhardy, @robotastronaut, @princefishthrower: you can do this too.)



> @MoranJustt: Everything looks good here. I am ready to merge this PR (at [d1af3b2](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d1af3b20ee0830e4a214dc94340bcc419af8d617)) on your behalf whenever you think it's ready.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! ❤️
> 
> (@ianhowe76, @pwrnrd, @CatGuardian, @MengRS, @Scoup, @isabela-morais, @rbutera, @peterblazejewicz, @danursin, @nhardy, @robotastronaut, @princefishthrower: you can do this too.)

Ready to merge

**Pull request merged**